src/dashboard/src/pages/Job/Tabs/Endpoints.tsx

```diff
@@ -41,7 +41,7 @@ const EndpointListItem: FunctionComponent<{ endpoint: any }> = ({ endpoint }) =>
   if (endpoint.name === 'ssh') {
     const identify = `${cluster['workStorage'].replace(/^file:\/\//i, '//')}/${job['jobParams']['workPath']}/.ssh/id_rsa`
     const host = `${endpoint['nodeName']}.${endpoint['domain']}`;
-    const task = job['jobParams']['jobtrainingtype'] === 'PSDistJob' ? endpoint['podName'].split('-').pop() : '';
+    const task = job['jobParams']['jobtrainingtype'] === 'PSDistJob' ? endpoint['podName'].split('-').slice(1).join('-') : '';
```

Comment:
Will this still work for clusters using `PythonLauncher`?

Reply:
No, shall we support both?

Reply:
As we agreed offline. @xudifsd please help add a label to each endpoint so that dashboard simply displays the label.

Reply:
See https://github.com/microsoft/DLWorkspace/pull/1165

Reply:
@xudifsd frontend changes align to #1165: [https://github.com/microsoft/DLWorkspace/pull/1156/commits/af0468a0d9adc299788aa344a1801c894386d908 ](https://github.com/microsoft/DLWorkspace/pull/1156/commits/af0468a0d9adc299788aa344a1801c894386d908)

Reply:
endpoints parsing commits will go on #1166